util/deltautil/apply.go

```diff
@@ -145,3 +132,27 @@ func WithDeltaManifest(m *pb.RunManifest, d *pb.Delta) (*pb.RunManifest, error)
 	}
 	return ret, nil
 }
+
+func ensureTargetExists(r *pb.RunManifest, targetID string) *pb.TargetManifest {
```

Comment:
I ran into these issues while testing. 
I'm not sure why these happen yet, but adding this fixed it.
```
Error(s) in logbus: 1 error occurred:
	* formatter: 6 errors occurred:
	* failed to handle delta manifest: target dd01c9fd-f9aa-4156-af62-31d7f8b56c55 not found in manifest
	* failed to handle delta manifest: target dd01c9fd-f9aa-4156-af62-31d7f8b56c55 not found in manifest
	* failed to handle delta manifest: target dd01c9fd-f9aa-4156-af62-31d7f8b56c55 not found in manifest
	* failed to handle delta manifest: target dd01c9fd-f9aa-4156-af62-31d7f8b56c55 not found in manifest
	* failed to handle delta manifest: target dd01c9fd-f9aa-4156-af62-31d7f8b56c55 not found in manifest
	* failed to handle delta manifest: target dd01c9fd-f9aa-4156-af62-31d7f8b56c55 not found in manifest
```
This case isn't hit every time.  I think we'll want to find out why this case is hit, but I also think t his is an ok patch until we figure out why.